dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/handler/ExternalSourceEntryItemUriListHandler.java

```diff
@@ -68,23 +61,16 @@ public boolean supports(List<String> uriList, String method) {
         return true;
     }
 
-    @Override
-    public boolean validate(Context context, HttpServletRequest request, List<String> uriList,
-                            Class clazz) throws AuthorizeException {
+    public boolean validate(Context context, HttpServletRequest request, List<String> uriList)
```

Comment:
I think you lost the `@Override` annotation here.

Reply:
This has been updated